client/app/containers/index.js

```diff
@@ -52,47 +54,51 @@ class App extends React.Component {
   }
 
   traverseToParent() {
-    const newPath = this.state.path.slice(0, this.state.path.lastIndexOf('/'));
-
-    this.pushNewState(newPath);
+    console.log(window.history.length)
+    window.history.back();
```

Comment:
Yes, you right. But this is old code. 
Also, that's why I replaced this code with this - https://github.com/sudodoki/offline-markdown-previewer/pull/26/commits/dbbfb680f31ff1a5e95bb9d86a068694486e6b10#diff-7f2cfb2ba71b1b9a920eb0144dd84aacR54